datahost-ld-openapi/src/tpximpact/datahost/ldapi/schemas/common.clj

```diff
@@ -22,18 +22,21 @@
                                                 false))))})
      :datahost/timestamp (let [utc-tz (ZoneId/of "UTC")]
                            (m/-simple-schema
-                            {:type :datahost/timestamp
+                             {:type :datahost/timestamp
                              :pred (fn timestamp-pred [ts]
                                      (and (instance? ZonedDateTime ts)
                                           (= (.getZone ^ZonedDateTime ts) utc-tz)))}))
-     
+
      ;; swagger-ui complains when namespaced keys are used,
      ;; so using non-namespaced
      :title-string (let [regex (str #"^\w[\w\s\D]+$")]
                      [:re {:error/message (format "Should match regex: %s" (str regex))} regex])
      ;; description allows newlines
      :description-string (let [regex #"^\w[\w\s\S\D]+$"]
                            [:re {:error/message (format "Should match regex: %s" (str regex))} regex])
+     :email-string (let [regex #"^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$"]
```

Comment:
"foo+bar@example.com" won't match but is a valid email address.

Also, we have TLDs like ".enterprises", ".technology" that won't fit into the 2-4 characters that this regex allows. I'd either use a library or relax the regex significantly (which would potentially allow invalid email addresses, but parsing those can be a rabbit hole and punting  is a valid strategy).

Reply:
Fair enough.. email regex parsing is notoriously impossible. I'm ok with loosening it all the way to {anything}@{anything}.{anything} for now if that seems reasonable to you?